src/scopes/activityReport/index.test.js

```diff
@@ -2411,9 +2411,34 @@ describe('filtersToScopes', () => {
         where: { id: [includedReport1.id, includedReport2.id, excludedReport.id] },
       });
     });
+  });
+
+  describe('delivery method', () => {
+    let includedReport1;
+    let includedReport2;
+    let excludedReport;
+    let possibleIds;
+
+    beforeAll(async () => {
+      includedReport1 = await ActivityReport.create({ ...draftReport, deliveryMethod: 'in-person' });
+      includedReport2 = await ActivityReport.create({ ...draftReport, deliveryMethod: 'in-person' });
+      excludedReport = await ActivityReport.create({ ...draftReport, deliveryMethod: 'hybrid' });
+      possibleIds = [
+        includedReport1.id,
+        includedReport2.id,
+        excludedReport.id,
+        globallyExcludedReport.id,
+      ];
+    });
+
+    afterAll(async () => {
+      await ActivityReport.destroy({
+        where: { id: [includedReport1.id, includedReport2.id, excludedReport.id] },
+      });
+    });
 
-    it('includes region id', async () => {
-      const filters = { 'region.in': ['2'] };
+    it('includes delivery method', async () => {
+      const filters = { 'deliveryMethod.in': ['in-person'] };
```

Comment:
I'd like to see these tests include 2 selections (for example: ```['in-person', 'hybrid']```) so the tests are a little more robust